sourced/ml/algorithms/token_parser.py

```diff
@@ -33,14 +33,16 @@ class TokenParser:
     # if True we have only ["sourc", "algorithm"].
     # if you do not want to filter small tokens set min_split_length=1.
     SAVE_TOKEN_STYLE = False  # whether yield metadata that can be used to reconstruct initial
-    # identifier.
+
+    # identifier
+    USE_NN = False  # wether to use neural network id splitter model instead of heuristic
```

Comment:
Can you also add another test to check TokenParser with use_nn enabled?

Reply:
I have a call with my university soon, no prob I'll do that just afterwards.
I should have fixed the rest, let's see if the test passes.